builder-20/eol-buildpack/buildpack.toml

```diff
@@ -0,0 +1,8 @@
+api = "0.9"
+
+[buildpack]
+  id = "heroku/eol-warning"
+  version = "1.0.0"
```

Comment:
A later iteration of the EOL buildpack for the shimmed builders added more metadata (so it would show up at `pack builder inspect` etc):
https://github.com/heroku/cnb-builder-images/commit/d513950262cf3173cc66be94b9b63e20485c8bc9

Would you mind adding similar `name`, `homepage` and `buildpack.licenses` here?

Reply:
https://github.com/heroku/cnb-builder-images/pull/552/commits/885a7643ba900ebe8ee748b30f646307ccc6d55e